app/models/account/syncable.rb

```diff
@@ -1,13 +1,42 @@
 module Account::Syncable
   extend ActiveSupport::Concern
 
+  included do
+    include AASM
+
+    enum :status, { ok: "ok", syncing: "syncing", error: "error" }, validate: true
```

Comment:
Coming from the model. It might be sensible to name this to `sync_state`. An `Account` may have other states (maybe related to the financial organization, such as `closed`, `unavailable`, etc.).